fusor-ember-cli/app/controllers/configure-environment.js

```diff
@@ -30,15 +30,15 @@ export default Ember.Controller.extend(ConfigureEnvironmentMixin, NeedsDeploymen
     selectEnvironment(environment) {
       this.set('showAlertMessage', false);
       this.set('selectedEnvironment', environment);
-      return this.get('deploymentController.model').set('lifecycle_environment', environment);
+      this.get('deploymentController.model').set('lifecycle_environment', environment);
     },
 
     createEnvironment(fields_env) {
       var self = this;
 
       var nameAlreadyExists =  self.get('lifecycleEnvironments').findBy('name', fields_env.name);
       if (nameAlreadyExists) {
-        return self.get('deploymentController').set('errorMsg', fields_env.name + ' is not a unique name. Environment not saved.');
+        self.get('deploymentController').set('errorMsg', envName + ' is not a unique name. Environment not saved.');
```

Comment:
ESLint would have caught this as an undef and failed travis, thanks for turning the rest of the rules on @cfchase!